examples/mpi/overview.ipynb

```diff
@@ -1465,6 +1467,7 @@
    ],
    "source": [
     "%%px\n",
+    "#NBVAL_IGNORE_OUTPUT\n",
```

Comment:
I (personally) find it very dangerous to add the flag to everything just to make CI happy as it was not necessary before.

Reply:
I agree with both comments. Just talked with Fabio about targeting to explore this issue.

Reply:
@mloubout is completely right. We need to find out what's going on. Who can we ask ? maybe the ipyparallel guys?